Does not recognize mp3 type

Hi! I have mp3 file what does not recognize by filetype. [Link to file](https://cloud.mail.ru/public/Mm2X/cNDSeJBnr)
First three bytes in my file: `0xff 0xfa 0xe3`
I have tried to use [python-magic](https://github.com/ahupp/python-magic) and it recognized this file like `audio/mpeg`. Can you look into it please?

You have the errors in the logs, just fix them and update the PR.

```
/home/runner/work/filetype.py/filetype.py/filetype/types/audio.py:46:5: E129 visually indented line with same indent as next logical line
/home/runner/work/filetype.py/filetype.py/filetype/types/audio.py:47:21: E117 over-indented
/home/runner/work/filetype.py/filetype.py/filetype/types/audio.py:55:5: E129 visually indented line with same indent as next logical line
/home/runner/work/filetype.py/filetype.py/filetype/types/audio.py:56:25: E117 over-indented
```

--- a/filetype/types/audio.py
+++ b/filetype/types/audio.py
@@ -54,12 +54,7 @@ class Mp3(Type):
                     buf[1] == 0xFA or #MPEG 1 with error protection
                     buf[1] == 0xFB):  #MPEG 1 w/o error protection
                     return True
-                else:
-                    return False
-            else:
-                return False
-        else:
-             return False
+        return False
 
 
 class M4a(Type):

--- a/filetype/types/audio.py
+++ b/filetype/types/audio.py
@@ -40,16 +40,24 @@ class Mp3(Type):
         )
 
     def match(self, buf):
-        return (len(buf) > 2 and
-                ((buf[0] == 0x49 and
-                  buf[1] == 0x44 and
-                  buf[2] == 0x33) or
-                 (buf[0] == 0xFF and
-                  buf[1] == 0xF2) or
-                 (buf[0] == 0xFF and
-                  buf[1] == 0xF3) or
-                 (buf[0] == 0xFF and
-                  buf[1] == 0xFB)))
+        if (len(buf) > 2):
+            if (buf[0] == 0x49 and
+                buf[1] == 0x44 and
+                buf[2] == 0x33):
+                return True
+
+            if (buf[0] == 0xFF):
+                if (buf[1] == 0xE2 or #MPEG 2.5 with error protection
+                    buf[1] == 0xE3 or #MPEG 2.5 w/o error protection
+                    buf[1] == 0xF2 or #MPEG 2 with error protection
+                    buf[1] == 0xF3 or #MPEG 2 w/o error protection
+                    buf[1] == 0xFA or #MPEG 1 with error protection
+                    buf[1] == 0xFB):  #MPEG 1 w/o error protection
+                    return True
+            else:
+                return False
+        else:
+             return False
 
 
 class M4a(Type):

--- a/filetype/types/audio.py
+++ b/filetype/types/audio.py
@@ -54,6 +54,8 @@ class Mp3(Type):
                     buf[1] == 0xFA or #MPEG 1 with error protection
                     buf[1] == 0xFB):  #MPEG 1 w/o error protection
                     return True
+                else:
+                    return False
             else:
                 return False
         else:

--- a/filetype/types/audio.py
+++ b/filetype/types/audio.py
@@ -44,16 +44,16 @@ class Mp3(Type):
             if (buf[0] == 0x49 and
                 buf[1] == 0x44 and
                 buf[2] == 0x33):
-                return True
+                    return True
 
             if (buf[0] == 0xFF):
-                if (buf[1] == 0xE2 or  #MPEG 2.5 with error protection
-                    buf[1] == 0xE3 or  #MPEG 2.5 w/o error protection
-                    buf[1] == 0xF2 or  #MPEG 2 with error protection
-                    buf[1] == 0xF3 or  #MPEG 2 w/o error protection
-                    buf[1] == 0xFA or  #MPEG 1 with error protection
-                    buf[1] == 0xFB):   #MPEG 1 w/o error protection
-                    return True
+                if (buf[1] == 0xE2 or  # MPEG 2.5 with error protection
+                    buf[1] == 0xE3 or  # MPEG 2.5 w/o error protection
+                    buf[1] == 0xF2 or  # MPEG 2 with error protection
+                    buf[1] == 0xF3 or  # MPEG 2 w/o error protection
+                    buf[1] == 0xFA or  # MPEG 1 with error protection
+                    buf[1] == 0xFB):   # MPEG 1 w/o error protection
+                        return True
         return False
 
 

--- a/filetype/types/audio.py
+++ b/filetype/types/audio.py
@@ -47,12 +47,12 @@ class Mp3(Type):
                 return True
 
             if (buf[0] == 0xFF):
-                if (buf[1] == 0xE2 or #MPEG 2.5 with error protection
-                    buf[1] == 0xE3 or #MPEG 2.5 w/o error protection
-                    buf[1] == 0xF2 or #MPEG 2 with error protection
-                    buf[1] == 0xF3 or #MPEG 2 w/o error protection
-                    buf[1] == 0xFA or #MPEG 1 with error protection
-                    buf[1] == 0xFB):  #MPEG 1 w/o error protection
+                if (buf[1] == 0xE2 or  #MPEG 2.5 with error protection
+                    buf[1] == 0xE3 or  #MPEG 2.5 w/o error protection
+                    buf[1] == 0xF2 or  #MPEG 2 with error protection
+                    buf[1] == 0xF3 or  #MPEG 2 w/o error protection
+                    buf[1] == 0xFA or  #MPEG 1 with error protection
+                    buf[1] == 0xFB):   #MPEG 1 w/o error protection
                     return True
         return False
 

